torch/_prims/__init__.py

```diff
@@ -1679,6 +1679,7 @@ def _split_dim_aten(a: Tensor, dim: int, outer_length: int) -> Tensor:
 # Note: allows dimensions to be specified redundantly
 def _squeeze_meta(a: TensorLikeType, dimensions: Sequence) -> TensorLikeType:
     assert isinstance(a, TensorLike)
+    dimensions = utils.canonicalize_dims(a.ndim, dimensions)
```

Comment:
The prims don't canonicalize dimensions, only the refs do

Reply:
The `expand_dims` prim calls `canonicalize_dims` for `unsqueeze`. There is an asymmetry here with the `squeeze` prim.